src/data/data.js

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
You can keep this as a .js file but in the future if you want you can convert this to .ts and declare the corresponding types